server/filters/formatQualificationLevelHintFilter.ts

```diff
@@ -0,0 +1,16 @@
+export default function formatQualificationLevelHintFilter(value: string): string {
```

Comment:
I thought we could use a separate filter for the "hints", but happy to canvas opinion on this.

Reply:
Yeah, I'm happy with this.
We don't have an equivalent of Gary's "contentLookup" , and I don't think we need one. I think thats what filters are for

We have other Induction pages that we have been migrating as part of this where we have hardcoded the hints in the template because we didnt have the corresponding filter etc, and at the time perhaps it was easier/quicker to just put the content in the template?

But I prefer this type of approach, and it's been there in the back of my mind that we should go back and write filters or similar for the hint text

This makes a good start and establishes that pattern. Good work 👍